spec/index.html

```diff
@@ -6575,8 +6475,8 @@ <h5>Strings in SPARQL Functions</h5>
               <h6>String arguments</h6>
               <p>Certain functions (e.g. <a href="#func-regex">REGEX</a>, <a href=
                                                                              "#func-strlen">STRLEN</a>, <a href="#func-contains">CONTAINS</a>) take a <code>string
-                  literal</code> as an argument and accept a simple literal, a plain literal with
-                language tag, or a literal with datatype xsd:string. They then act on the lexcial form
+                  literal</code> as an argument and accept a simple literal, or a literal with
+                language tag. They then act on the lexical form
```

Comment:
```suggestion
                  literal</code> as an argument and accept a simple literal,
                  a literal with datatype xsd:string, or a literal with
                language tag. They then act on the lexical form
```

Reply:
In fact, I am not even sure whether simple literals should be mentioned here. As I wrote in one of my other comments above, simple literals are something that is specific in the context of concrete syntaxes as syntactic sugar for typed literals with datatype xsd:string (and no language tag). I think that, for the definition of the functions, the syntactic sugar is irrelevant.

Reply:
Post first pass review: maybe we do need some language, not terminology to be able to refer to "xsd:string or rdf:langString" for use in the function definitions where that case occurs frequently.

"string literal" is the best I can come up with for now - it is a bit close to "xsd:string".

A longer phrase "string or language tagged literal" somewhat defeats the point of having such language. Maybe we just have to write "xsd:string or rdf:langString".

Reply:
Agree. I think that "string literal" is a good term for this purpose.

Reply:
I agree with the term "string literal".